README.md

```diff
@@ -894,16 +894,10 @@ Extensions
 Module json4s-ext contains extensions to extraction and serialization. The following types are supported.
 
 ```scala
-// Lift's box
-implicit val formats = org.json4s.DefaultFormats + new org.json4s.native.ext.JsonBoxSerializer
-
 // Scala enums
 implicit val formats = org.json4s.DefaultFormats + new org.json4s.ext.EnumSerializer(MyEnum)
 // or
 implicit val formats = org.json4s.DefaultFormats + new org.json4s.ext.EnumNameSerializer(MyEnum)
-
-// Joda Time
-implicit val formats = org.json4s.DefaultFormats ++ org.json4s.ext.JodaTimeSerializers.all
```

Comment:
🤔 ???

https://github.com/json4s/json4s/blob/af981779e05eb46f5675320f431ae2f620894b6c/ext/src/main/scala/org/json4s/ext/JodaTimeSerializers.scala#L25

Reply:
oh..sorry, 

This deletion is completely incorrect. 
Only the lift's Box should have been removed.